### ISSUE TITLE
custom URL unable to resolve host

### ISSUE THREAD

**Issue by keepcarlson** [mendhak/gpslogger#773]
Hi, trying to use ESRI ArcGIS GeoEvent Server to receive data from GPS Logger.  Custom URL from that server is https://parcgisappcit1.ad.brown.edu:6143/geoevent/rest/receiver/GPS_Logger_In_JSON to receive json on a rest endpoint

Getting error unable to resolve host .... no address associated with hostname  
android_getaddrinfo failed: EAI_NODATA  

Can't validate SSL either.  

error messages and debug file attached.

Had great luck with using these two applications previously, so I know they work.....had to move to a new server  :-(  

I don't know if you can help me trouble shoot ideas about what could be going wrong?   I have  a research project coming  up.  Thanks so much.

[debuglog.txt](https://github.com/mendhak/gpslogger/files/3561142/debuglog.txt)
![Screenshot_20190830-141718_GPSLogger](https://user-images.githubusercontent.com/5710167/64044435-c677f900-cb34-11e9-862b-b6ddd648844a.jpg)
![Screenshot_20190830-142353_GPSLogger](https://user-images.githubusercontent.com/5710167/64044438-c7a92600-cb34-11e9-95f8-2f726b6206e7.jpg)

**Comment by mendhak** [comment]
I tried hitting that URL but I'm also not able to see it.  I tried a few other commands, but nothing is resolving that DNS to an IP address.  

![image](https://user-images.githubusercontent.com/746276/64061554-5751e180-cbd4-11e9-9c2f-c5b9b2629024.png)

I did some searching and it appears that `ad.brown.edu` is  an Active Directory server in the domain Brown.edu.  So this could very well be private DNS that only works if you're connected to the Brown.edu network.   

My guess would be, if you connect to the brown.edu network and try this again, the app will be able to connect.  However if you leave that network, Custom URL logging will fail as that address is not resolvable. 

If you are looking to use the `parcgisappcit1.ad.brown.edu` domain from outside the Brown.edu network you will need to ask your administrators to make that service reachable from the public Internet; for example they may provide an alternate address which proxies through to the target.
